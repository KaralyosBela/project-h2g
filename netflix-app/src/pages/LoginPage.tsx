import { Layout } from "../ui/Layout";
import { LoginCard } from "../components/LoginCard";
import { AddMovieModal } from "../components/AddMovieModal";
import { EditMovieModal } from "../components/EditMovieModal";
export const LoginPage: React.FC = () => {
  return (
    <div>
      <Layout>
        {/* <AddMovieModal/> */}
        {/* <EditMovieModal/> */}
        <LoginCard />
      </Layout>
    </div>
  );
};
