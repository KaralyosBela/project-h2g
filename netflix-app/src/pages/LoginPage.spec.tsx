import React from "react";
import { render } from "@testing-library/react";
import { LoginPage } from "./LoginPage";

jest.mock("../ui/Layout", () => ({
  Layout: () => <div>Container</div>,
}));
//komponens nÃ©v
describe("LoginPage", () => {
  it("should render component", () => {
    const {container} = render(<LoginPage />);
    expect(container).toMatchSnapshot();
  });
});
