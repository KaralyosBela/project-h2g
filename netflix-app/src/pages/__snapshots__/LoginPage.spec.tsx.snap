// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage should render component 1`] = `
<div>
  <div>
    <div>
      Container
    </div>
  </div>
</div>
`;
